sdk/core/core-http/src/policies/bearerTokenAuthenticationPolicy.ts

```diff
@@ -240,6 +240,12 @@ export function bearerTokenAuthenticationPolicy(
     }
 
     public async sendRequest(webResource: WebResourceLike): Promise<HttpOperationResponse> {
+      if (!webResource.url.toLowerCase().startsWith("https")) {
+        throw Error(
```

Comment:
nit: `throw Error` or `throw new Error`? I see both in our code base, latter is majority

Reply:
true! Thank you